sycl/include/sycl/ext/oneapi/matrix/matrix-jit.hpp

```diff
@@ -737,6 +738,160 @@ class wi_element<uint16_t, NumRows, NumCols, Layout, Group> {
   }
 };
 
+template <size_t NumRows, size_t NumCols, matrix_layout Layout, typename Group>
+class wi_element<sycl::ext::intel::experimental::bfloat16, NumRows, NumCols,
```

Comment:
Nit: probably alias like
```
using bfloat16 = sycl::ext::intel::experimental::bfloat16
```
could improve readability of this code.

Reply:
we can't using "using" in header file

Reply:
> we can't use "using" in header file

What do you mean?